azurerm/resource_arm_redis_cache.go

```diff
@@ -473,6 +486,7 @@ func resourceArmRedisCacheRead(d *schema.ResourceData, meta interface{}) error {
 	if props := resp.Properties; props != nil {
 		d.Set("ssl_port", props.SslPort)
 		d.Set("hostname", props.HostName)
+		d.Set("minimum_tls_version", props.MinimumTLSVersion)
```

Comment:
Should we be converting this to a string?
```suggestion
		d.Set("minimum_tls_version", string(props.MinimumTLSVersion))
```